docs/pages/enroll-resources/database-access/enroll-self-hosted-databases/oracle-exadata.mdx

```diff
@@ -0,0 +1,428 @@
+Service:
+```code
+$ sudo teleport db configure create \
+   -o file \
+   --token=/tmp/token \
+   --proxy=<Var name="example.teleport.sh:443" /> \
+   --name="<Var name="oracle" />" \
+   --protocol=oracle \
+   --uri="<Var name="10.20.30.40:2484" />" \
+   --ca-cert-file="var/lib/teleport/oracle-server-certificate.crt" \
+   --labels=env=dev
+```
+
+The final entry for the `<Var name="oracle" />` database will look similar to this:
+
+```yaml
+db_service:
+  enabled: true
+  databases:
+  - name: <Var name="oracle" />
+    uri: "<Var name="10.20.30.40:2484" />"
+    protocol: oracle
+    tls:
+      mode: verify-ca
+      ca_cert_file: /var/lib/teleport/oracle-server-certificate.crt
+```
+
+(!docs/pages/includes/start-teleport.mdx service="the Teleport Database Service"!)
+
+</TabItem>
+<TabItem label="Kubernetes Cluster">
+Create a secret containing the database CA certificate in the same namespace
+as Teleport using the following command:
+
+```code
+$ kubectl create secret generic db-ca --from-file=ca.pem=/path/to/oracle-server-certificate.crt
+```
+
+Create a file called `values.yaml` with the following content. Update <Var
+name="example.teleport.sh" /> to use the host and port of the Teleport Proxy
+Service and <Var name="JOIN_TOKEN" /> to the join token you created earlier using the `tctl tokens add` command.
+
+```yaml
+roles: db
+proxyAddr: <Var name="example.teleport.sh" />
+enterprise: true
+authToken: "<Var name="JOIN_TOKEN" />"
+databases:
+  - name: <Var name="oracle" />
+    uri: "<Var name="10.20.30.40:2484" />"
+    protocol: oracle
+    static_labels:
+      env: dev
+    tls:
+      mode: verify-ca
+      ca_cert_file: /etc/teleport-tls-db/db-ca/ca.pem
+extraVolumes:
+  - name: db-ca
+    secret:
+      secretName: db-ca
+extraVolumeMounts:
+  - name: db-ca
+    mountPath: /etc/teleport-tls-db/db-ca
+    readOnly: true
+```
+
+Teleport provides Helm charts for installing the Teleport Database Service in Kubernetes Clusters.
+
+(!docs/pages/includes/kubernetes-access/helm/helm-repo-add.mdx!)
+
+Install the chart:
+
+   ```code
+   $ helm install teleport-kube-agent teleport/teleport-kube-agent \
+     --create-namespace \
+     --namespace teleport-agent \
+     --version (=teleport.version=) \
+     -f values.yaml
+   ```
+
+Make sure that the Teleport agent pod is running. You should see one
+`teleport-kube-agent` pod with a single ready container:
+
+   ```code
+   $ kubectl -n teleport-agent get pods
+   NAME                    READY   STATUS    RESTARTS   AGE
+   teleport-kube-agent-0   1/1     Running   0          32s
+   ```
+
+</TabItem>
+</Tabs>
+
+
+## Step 5/6. (Optional) Configure Teleport to pull audit logs from Oracle Audit Trail
+
+Teleport can pull audit logs from Oracle Audit Trail.
+In order to enable this feature, you will need to configure Oracle Audit Trail and create a dedicated Teleport
+user that will be used to fetch audit events from Oracle Audit Trail.
```

Comment:
create new include and share between the oracle guides?

Reply:
Not sure if its worth it at this moment? We only have two guides, and this section doesn't match 1-1 with the other one as the variables are different here, as we are using IP addresses in this one.

Reply:
I'm leaving it as it is for now.